lib/Cron/Workers/StatsNotifications/StatsNotificationsRepository.php

```diff
@@ -15,20 +15,17 @@ protected function getEntityClassName() {
 
   /**
    * @param int $newsletter_id
-   * @return StatsNotificationEntity[]
+   * @return StatsNotificationEntity|null
    */
   public function findByNewsletterId($newsletter_id) {
```

Comment:
I see the logic changed from `Entity[]` to `Entity|null`. Then I'd suggest naming it `findOneByNewsletterId` or sth more specific if it would make sense (like `findFirst...`, `findLast...`, `findLatest...`, etc.).